src/seed_tools/commands/create_seed.ts

```diff
@@ -13,69 +13,165 @@ import {
   type Study,
 } from '../../proto/generated/study';
 import { VariationsSeed } from '../../proto/generated/variations_seed';
+import diffStrings from '../utils/diff_strings';
+import * as file_utils from '../utils/file_utils';
 import * as seed_validation from '../utils/seed_validation';
 import * as study_json_utils from '../utils/study_json_utils';
-import * as study_validation from '../utils/study_validation';
-
-export default new Command('create_seed')
-  .description('Create seed.bin from study files')
-  .argument('<studies_dir>', 'path to a directory containing study files')
-  .argument('<output_file>', 'output seed file')
-  .option('--version <value>', 'version to set into the seed')
-  .option(
-    '--serial_number_path <path>',
-    'file path to write the serial number to',
-    './serialnumber',
-  )
-  .option('--mock_serial_number <value>', 'mock serial number')
-  .action(main);
+
+export default function createCommand() {
+  return new Command('create_seed')
+    .description('Create seed.bin from study files')
+    .argument('<studies_dir>', 'path to the directory containing study files')
+    .argument(
+      '[output_seed_file]',
+      'output seed file, may be omitted if --validate_only is provided',
+    )
+    .option('--fix', 'fix format errors in-place')
+    .option('--mock_serial_number <value>', 'mock serial number')
+    .option(
+      '--output_serial_number_file <path>',
+      'file path to write the seed serial number',
+      './serialnumber',
+    )
+    .option('--validate_only', 'validate the seed without creating it')
```

Comment:
The name sounds confusing to me with a combination of `--fix`.
What if we just remove it and use `output_seed_file == null` instead?

Reply:
fixed by introducing separate `create` and `lint` commands.